cmake/Modules/FindMKLDNN.cmake

```diff
@@ -126,6 +125,9 @@ IF(NOT TARGET mkldnn)
 ENDIF(NOT TARGET mkldnn)
 IF(MKL_FOUND)
   TARGET_COMPILE_DEFINITIONS(mkldnn PRIVATE -DUSE_MKL)
+  IF(USE_CBLAS_IN_MKLDNN)
```

Comment:
To make the USE_CBLAS option only effective for MKL-DNN, I change the add_definition to "TARGET_COMPILE_DEFINITIONS", so that it will not influence the other modules.